django-reference-app/django-ref-config/wsgi.py

```diff
@@ -11,6 +11,6 @@
 
 from django.core.wsgi import get_wsgi_application
 
-os.environ.setdefault("DJANGO_SETTINGS_MODULE", "djangoapp.settings")
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "clusterapp.settings")
```

Comment:
Is the name of the settings module correct here? It looks like the file is actually called `django-ref-config.settings`

Reply:
Good catch, was not correct. Legacy app name, which has been corrected in `
e9ff24a`.